python_on_whales/components/container/cli_wrapper.py

```diff
@@ -1707,7 +1707,11 @@ def start(
         else:
             run(full_cmd)
 
-    def stats(self, all: bool = False) -> List[ContainerStats]:
+    def stats(
+        self,
+        containers: Union[ValidContainer, List[ValidContainer]] = [],
```

Comment:
Default should be None. [] should return [] as output without executing any command.